### DISCUSSION TITLE
Text Animation presets

### DISCUSSION THREAD

**Discussion by smnaveen96** [keiwando/toonsquid#74] [Ideas]
Hi, @keiwando i’m not an animator… i just randomly saw this Toonsquid app somewhere on internet..it was quite interesting and gone through your discussion/introduction about Toonsquid on reddit..and i wanted to give it a shot so bought this and Evolution to support your Hard work and improve my skills as well.. Here is my small request.. Since toonsquid supports importing videos we can use it for editing small videos like 30secs right? So it would be nice if you include text animation presets like in Canva..is that possible?

**Comment by keiwando** [comment]
Thank you very much, I appreciate your support!
Yes, I would definitely like to add effect presets including text animations to ToonSquid eventually but there are still a few structural things that I need to figure out first before I can add this feature in the way that I want.
